mappings/net/minecraft/advancement/criterion/CriterionProgress.mapping

```diff
@@ -5,7 +5,7 @@ CLASS net/minecraft/class_178 net/minecraft/advancement/criterion/CriterionProgr
 	METHOD method_784 isObtained ()Z
 	METHOD method_785 fromPacket (Lnet/minecraft/class_2540;)Lnet/minecraft/class_178;
 		ARG 0 buf
-	METHOD method_786 getObtainedDate ()Ljava/util/Date;
+	METHOD method_786 getObtained ()Ljava/util/Date;
```

Comment:
Hmm, this might be confused with `isObtained` above. I'd rather rename the field to `obtainedDate`.

Reply:
I think the toString of this class was like `"obtained: " + (obtained == null ? "false" : obtained)` and the field hence named.

Reply:
Yeah the toString that it's "obtained", but I agree with Juuz, obtainedDate is less confusing